### ISSUE TITLE
Use Go 1.13's new wrapped error support

### ISSUE THREAD

**Issue by mperham** [go-bindata/go-bindata#51]
These `Errorf` calls could take advantage of the new `%w` support in 1.13 so the underlying error can still be accessed and queried easily, just change %v to %w.

```
webui/static.go:		return nil, fmt.Errorf("Read %q: %v", name, err)
webui/static.go:		return nil, fmt.Errorf("Read %q: %v", name, err)
webui/static.go:			return nil, fmt.Errorf("Asset %s can't read by error: %v", name, err)
webui/static.go:			return nil, fmt.Errorf("AssetInfo %s can't read by error: %v", name, err)
```

I'm not sure your policy on supporting older Go versions but perhaps go-bindata could only emit this code if running in Go 1.13+.

**Comment by kevinburke** [comment]
Good idea, just fixed this here. https://github.com/kevinburke/go-bindata/commit/c938b041d272dc5b5ad53b4884cbe59083cc0359

(The red "X" is because tests on Go 1.12 now generate a different file than Go 1.13; fixed in a later commit.)

**Comment by mperham** [comment]
@kevinburke Neat trick with the conditional build. Forgot about that!
